swip/commands/add.py

```diff
@@ -0,0 +1,73 @@
+import shutil
+from pathlib import Path
+
+import common.utils as utils
+from common.paths import PATHS as path_to
+from common.utils import logger
+
+
+def validate_path(input_path):
```

Comment:
Using `mypy --strict` will help you find where you forgot to put type annotations :)